### PR TITLE
Option flag to set leasing amount threshold added

### PR DESCRIPTION
**Pull request by alexeykiselev**

### CODE REVIEW

File: lessor.go [hunk 1]
```diff
@@ -75,6 +75,7 @@ func run() error {
 		lessorPK            string
 		leasingAddress      string
 		irreducibleBalance  int64
+		leasingThreshold    int64
```
Comment (esuwu) [inline lessor.go:78]: Is it okay for `leasingThreshold` be negative?

File: lessor.go [hunk 2]
```diff
@@ -343,6 +345,12 @@ func run() error {
 		log.Print("[ERROR] Negative of zero amount to lease")
 		return errFailure
 	}
+	if leasingThreshold > 0 {
```
Comment (esuwu) [inline lessor.go:348]: If you are going to keep leasingThreshold `integer`, should we check `if leasingThreshold < 0` ?

File: lessor.go (hunk 1 above)
Comment (alexeykiselev) [reply lessor.go:78]: The negative value is useless and must be ignored.

File: lessor.go (hunk 2 above)
Comment (alexeykiselev) [reply lessor.go:348]: Nope, only positive values are meaningful.

**Review by esuwu — APPROVED** [review]

**Pull request merged**